### DISCUSSION TITLE
Using `pagedown::chrome_print()` to convert html slides to PDF results in indentation of code blocks

### DISCUSSION THREAD

**Discussion by snhansen** [quarto-dev/quarto-cli#9848] [Q&A] [answered]
### Description

Consider this simple reveal.js slide made with Quarto:

	---
	format: revealjs
	---
	## Section

	```{r}
	#| echo: true
	x <- 1:10
	sum(x)
	```
Suppose I want to convert the resulting .html to a .pdf. We could use `pagedown::chrome_print()` or the method described in the [documentation](https://quarto.org/docs/presentations/revealjs/presenting.html#print-to-pdf). Both methods leads to a rather large indentation of code blocks. I don't know if this something that's related to how Quarto produces the .html or if it's just how printing with Chrome works. In any case, I'm interested in removing this indentation. Any ideas?

Here's a screenshot of the .html:

![image](https://github.com/quarto-dev/quarto-cli/assets/45770265/5cd32213-a436-46ed-83c1-ebfaaea36bdf)

Here's a screenshot of the .pdf:

![image](https://github.com/quarto-dev/quarto-cli/assets/45770265/38acfeff-e9b0-4f58-b7a5-a7110e159080)

I'm on Windows and this is made using Quarto 1.4.554, pagedown v. 0.20 and Chrome v. 125.0.6422.142.

**Comment by snhansen** [comment]
I did some investigation and it seems to be the `sourceCode` class applied at the code element:

![image](https://github.com/quarto-dev/quarto-cli/assets/45770265/182fcfc4-aa17-42c4-9fcf-0f686df2bae5)

Indeed if I remove the `sourceCode` class from this line, the indentation is gone when printing to .pdf.

**Comment by snhansen** [comment]
I just checked that it's the same behavior on macOS. Here's a link to the .html slide used above if anyone is interested in seeing if they can replicate the issue and/or find a solution: https://biostat.au.dk/stefan/pdf-print/test.html

**Comment by mcanouil** [comment, marked as answer]
Following the documentation (thus the print to PDF method integrated in Reveal.js) and using Quarto 1.5.

<img width="1624" alt="image" src="https://github.com/quarto-dev/quarto-cli/assets/8896044/67f1c7d3-a718-45f3-b234-76456b1c105a">

As you can see I cannot replicate.
But I can using the same version as in your document.
Maybe a Reveal.js bug fixed with an update.

Anyhow, it seems fixes on `main` (1.5).

**Reply by snhansen** [reply]
Yeah, I just updated to 1.5.43 and it is indeed fixed there. Should've tried that earlier, geez. Thanks for the feedback -- much appreciated.
